src/version.ts

```diff
@@ -0,0 +1,23 @@
+export const DEFAULT_VERSION: DotNetFormatVersion = "3";
+
+export type DotNetFormatVersion =
+  | "3"
+;
+
+const supportedVersions: DotNetFormatVersion[] = [
+  "3",
+];
+
+export function checkVersion(version: string): DotNetFormatVersion {
+  // if no version is supplied default to 3 since that was the first supported version
+  if (!version) return DEFAULT_VERSION;
```

Comment:
In `action.yml` we're setting the default `version` to `3` so this check isn't needed.